### PR TITLE
Module 1: First Capstone Project

### PR DESCRIPTION
**Pull request by ikennarichard**
## Module 1 (Week 5): First Capstone Project - Concert Page
For this project, I
- Used the HTML, CSS and JavaScript linters to check for consistency
- Added the Home and About page.
- Implemented responsive design with media queries.
- Added mobile navigation menu.
- Added the hover effects on the program cards.
- 'More' button on mobile displays the list of artists by increasing the parent's height
- Featured artists section was added dynamically through JavaScript.
- Added the README file including details about the project
-  Video [Details](https://www.loom.com/share/8837aa04609e4ad981fd98460b107e86) of the project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+          <nav class="navbar">
+            <ul>
+              <li><a href="./about.html">About</a></li>
+              <li><a href="#artists">Artists</a></li>
+              <li><a href="#conatct">Contact</a></li>
+              <li><a href="#news">News</a></li>
+              <li><a href="#livestreams">LiveStreams</a></li>
+            </ul>
+              </nav>
+              <a href="#" class="header_tickets">Get Tickets</a>
```
Comment (mukhammadrizooff) [inline index.html:33]: - [ ] [REQUIRED] Good job so far! Kindly remove necessary space between second navigation elements like a [Figma Design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    </section>
+
+    <section class="featured_artists" id="artists">
+      <h2 class="heading">Featured Artists</h2>
+      <i class="red_line"></i>
+      <div class="container">
+
+      </div>
+      <button type="button" class="more_button">More <img src="./icons/down-arrow.svg" alt="" class="dropdown"></button>
+    </section>
```
Comment (mukhammadrizooff) [inline index.html:93]: - [ ] [REQUIRED] Good job so far! Please add also `chess`(you can choose optional to set color of chess shape) background shape to each artist's picture like a [Figma Design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 

image

File: about.css [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+  position: absolute;
+  top: 0;
+  transform: translate(50%, 50%);
+  width: 50%;
+  cursor: pointer;
+  transition: text-shadow 0.3s ease-in-out;
+  color: #fff;
+  line-height: 1.5;
+  font-weight: bold;
+}
```
Comment (mukhammadrizooff) [inline about.css:133]: - [ ] [REQUIRED] Good job so far! Please add `picture` about your project to background of `card` like a [Figma Design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @ikennarichard, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Correct Github Flow ✔️ 
- Good implementations ✔️ 
- Descriptive `PR` summary ✔️ 
- Descriptive `Commit` history ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.css (hunk 3 above)
Comment (ikennarichard) [reply about.css:133]: Good evening, please 🙏 can you check again, the pictures are showing on my browser. I'm so confused @mukhammadrizooff

**Review by omarsalem7 — APPROVED** [review]
Hi @ikennarichard 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Made all changes requested by the previous reviewer
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**